gdx/src/com/badlogic/gdx/graphics/g2d/BitmapFont.java

```diff
@@ -844,6 +910,9 @@ public void getGlyphs (GlyphRun run, CharSequence str, int start, int end, Glyph
 				if (ch == '\r') continue; // Ignore.
 				Glyph glyph = getGlyph(ch);
 				if (glyph == null) {
+					if (Character.isHighSurrogate(ch)) {
+						continue;
+					}
```

Comment:
Minor but ideally brace style is consistent within a file, in this case "no brace for single line statement".